XPlat/ExposureNotification/source/Xamarin.ExposureNotification/CallbackService.android.cs

```diff
@@ -40,25 +40,66 @@ class ExposureNotificationCallbackService : JobIntentService
 	{
 		const int jobId = 0x02;
 
-		public static void EnqueueWork(Context context, Intent work)
+		public static void EnqueueWork(Context? context, Intent? work)
 			=> EnqueueWork(context, Java.Lang.Class.FromType(typeof(ExposureNotificationCallbackService)), jobId, work);
 
 		protected override async void OnHandleWork(Intent workIntent)
 		{
-			var token = workIntent.GetStringExtra(ExposureNotificationClient.ExtraToken);
+			// Not really "obsolete" as this is just Google's recommendation
+			// and v1 might still be the only thing on the device.
+#pragma warning disable CS0618
 
-			var summary = await ExposureNotification.PlatformGetExposureSummaryAsync(token);
+			var token = workIntent.GetStringExtra(ExposureNotificationClient.ExtraToken);
+			if (token == null)
+				return;
 
-			Task<IEnumerable<ExposureInfo>> GetInfo()
+			// we used windows mode and implemented the windows mode handler
+			if (token == ExposureNotificationClient.TokenA && ExposureNotification.DailySummaryHandler != null)
```

Comment:
It is been a while since this PR was made, and Google apparently went out from Token concept ([link](https://developers.google.com/android/reference/com/google/android/gms/nearby/exposurenotification/ExposureNotificationClient#public-static-final-string-extra_token), [link2](https://developers.google.com/android/reference/com/google/android/gms/nearby/exposurenotification/ExposureNotificationClient#public-static-final-string-token_a)).
Now, they are recommending to use token-less version of the API. Would you consider changing this?
<img width="860" alt="image" src="https://user-images.githubusercontent.com/14860255/110818544-a7ac0400-828d-11eb-8a79-7295b2df8132.png">